libs/cardano-ledger-test/src/Test/Cardano/Ledger/Examples/BabbageFeatures.hs

```diff
@@ -53,10 +52,6 @@ import GHC.Stack
 import Lens.Micro
 import qualified PlutusLedgerApi.V1 as PV1
 import Test.Cardano.Ledger.Alonzo.Scripts (alwaysSucceeds)
-import Test.Cardano.Ledger.Alonzo.Tools (
-  exampleExUnitCalc,
-  exampleInvalidExUnitCalc,
```

Comment:
Could the definitions of these functions also be removed?

Reply:
Thank you! Yes, the whole module can be removed, in fact. I kept it around for reference while I was working on the new implementation, but forgot to delete it at the end.